src/pico/i_video.c

```diff
@@ -879,7 +727,7 @@ void __noinline new_frame_init_overlays_palette_and_wipe() {
                         g = gammatable[usegamma-1][g];
                         b = gammatable[usegamma-1][b];
                     }
-                    palette[i] = PICO_SCANVIDEO_PIXEL_FROM_RGB8(r, g, b);
+                    palette[i] = PICO_SCANVIDEO_PIXEL_FROM_RGB8(b, g, r); // BOB: don't ask I have no idea
```

Comment:
Many screens support either BGR or RGB, so we could change this back.

Also if you were wondering why pico-scanvideo is still included, we need `PICO_SCANVIDEO_PIXEL_FROM_RGB` - though we could just copy the defs somewhere